README.md

```diff
@@ -11,6 +11,10 @@
 <a href="https://oras.land/"><img src="https://oras.land/img/oras.svg" alt="banner" width="100px"></a>
 </p>
 
+`oras-credentials-go` is a credential management library designed for [`oras-go`](https://github.com/oras-project/oras-go). It supports reading, saving, and removing credentials from Docker configuration files and external credential stores that follow the [Docker credential helper protocol](https://docs.docker.com/engine/reference/commandline/login/#credential-helper-protocol). It does not handle credential encryption and decryption.
```

Comment:
The sentence
> It does not handle credential encryption and decryption

is bit confusing. Rewording requested.

Reply:
Could you elaborate a bit on the confusing part?

Reply:
Let's just remove this sentence then.